go/vt/sqlparser/ast.go

```diff
@@ -745,8 +762,122 @@ type (
 
 	// IndexType is the type of index in a DDL statement
 	IndexType int8
+
+	// HandlerAction is the type of action for the DeclareHandler statement
+	HandlerAction int8
 )
 
+// Compound Statements
+type (
+	// CompoundStatements represents a list of compound statements.
+	CompoundStatements []CompoundStatement
```

Comment:
If possible, we should try to avoid AST types that are bare slices. They come with some issues that are nice to avoid. If we could use a struct containing the slice, I'd be happy.

Reply:
Fixed